resource/reapi/bindings/c/reapi_module.h

```diff
@@ -60,7 +61,7 @@ void reapi_module_destroy (reapi_module_ctx_t *ctx);
  *                   the match operation.
  *  \return          0 on success; -1 on error.
  */
-int reapi_module_match (reapi_module_ctx_t *ctx, const char *match_op,
+int reapi_module_match (reapi_module_ctx_t *ctx, enum match_op_t match_op,
```

Comment:
No need for `enum` type in function signature.
```suggestion
int reapi_module_match (reapi_module_ctx_t *ctx, match_op_t match_op,
```